catalog/plugin/catalog-plugin-metacardbackup-filestorage/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/storage/filestorage/MetacardFileStorageRoute.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.plugin.metacard.backup.storage.filestorage;
+
+import static org.apache.camel.builder.PredicateBuilder.not;
+
+import java.util.Map;
+import java.util.UUID;
+
+import org.apache.camel.CamelContext;
+import org.apache.commons.lang3.StringUtils;
+import org.codice.ddf.catalog.plugin.metacard.backup.common.MetacardStorageRoute;
+import org.codice.ddf.catalog.plugin.metacard.backup.common.MetacardTemplateBean;
+import org.codice.ddf.catalog.plugin.metacard.backup.common.ResponseMetacardActionSplitter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MetacardFileStorageRoute extends MetacardStorageRoute {
+    public static final String OUTPUT_PATH_TEMPLATE = "outputPathTemplate";
+
+    protected String outputPathTemplate;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MetacardFileStorageRoute.class);
+
+    public MetacardFileStorageRoute(CamelContext camelContext) {
+        super(camelContext);
+    }
+
+    public String getOutputPathTemplate() {
+        return outputPathTemplate;
+    }
+
+    public void setOutputPathTemplate(String outputPathTemplate) {
+        this.outputPathTemplate = outputPathTemplate;
+    }
+
+    @Override
+    public void configure() throws Exception {
+        String metacardRouteId = "metacard-" + UUID.randomUUID().toString();
```

Comment:
I'm going to leave as-is since this usage is internal only to the class to ensure route names are unique and the dashes (present or not) doesn't matter.